### PR TITLE
fix(cc): no classes styling the same CSS properties are set on one element

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
## Description
This PR ensures no classes styling the same CSS properties are being set on the same HTML element.

**BREAKING CHANGE:**
- Expandable info prop has been deprecated and removed

## Testing
Tested the changes in Chrome, Safari and Firefox

### CODE REVIEW

**Review by imprashast — APPROVED** [review]

File: packages/button/src/component.tsx [hunk 1]
```diff
@@ -15,73 +15,63 @@ import type { ButtonProps } from './props.js';
 const buttonVariants = ['primary', 'secondary', 'negative', 'utility', 'pill', 'link'] as const;
 
 export const Button = forwardRef<HTMLButtonElement | AnchorHTMLAttributes<HTMLAnchorElement>, ButtonProps>((props, ref) => {
-  const { primary, secondary, negative, utility, quiet, small, link, pill, loading, fullWidth, ...rest } = props;
+  const { primary, secondary, negative, utility, quiet, small, link, href, pill, loading, disabled, fullWidth, ...rest } = props;
```
Comment (BalbinaK) [inline packages/button/src/component.tsx:18]: Not sure which change caused this but now you can tab to a disabled button, which was not possible before and I think should not be possible.
Screenshot of an outlined disabled button

**Review by BalbinaK — CHANGES_REQUESTED** [review]

File: packages/button/src/component.tsx (hunk 1 above)
Comment (felicia-haggqvist) [reply packages/button/src/component.tsx:18]: Good catch! I'll look into it! :)

Comment (felicia-haggqvist) [reply packages/button/src/component.tsx:18]: Updated in this commit: [78a2d55](https://github.com/warp-ds/react/pull/274/commits/78a2d550d475e34a957476b450aa330242483305)

File: packages/steps/src/component.tsx [hunk 2]
```diff
@@ -9,24 +9,14 @@ export const StepsContext = createContext<{
   horizontal?: boolean;
   right?: boolean;
 }>({
-  horizontal: undefined,
-  right: undefined,
+  horizontal: false,
+  right: false,
 });
 
-export function Steps({ horizontal, right, className, children }: StepsProps) {
```
Comment (BalbinaK) [inline packages/steps/src/component.tsx:16]: Was it intended to remove className prop from the array in line 19?

Comment (felicia-haggqvist) [reply packages/steps/src/component.tsx:16]: Oh no! That was not intentional! 😄 🙈  I'll fix it! 😄

Comment (felicia-haggqvist) [reply packages/steps/src/component.tsx:16]: Updated in this commit: [233d18d](https://github.com/warp-ds/react/pull/274/commits/233d18da5e607b96df723668705f8bfdf160b5a0)

File: packages/attention/src/component.tsx [hunk 3]
```diff
@@ -185,13 +183,11 @@ export function Attention(props: AttentionProps) {
...
-                id: 'attention.aria.close',
-                message: 'Close',
-                comment: 'Aria label for the close button in attention',
-              },
-            )}
+            aria-label={i18n._({
+              id: 'attention.aria.close',
+              message: 'Close',
+              comment: 'Aria label for the close button in attention',
+            })}
```
Comment (BalbinaK) [inline packages/attention/src/component.tsx:190]: I'm not sure this makes any difference in production or if it's picked up by Crowdin after all, but we could run `pnpm messages:extract && pnpm messages:compile` when updating translation logic in the components.

File: packages/attention/src/component.tsx [hunk 4]
```diff
@@ -164,17 +172,7 @@ export function Attention(props: AttentionProps) {
...
-          invisible: !isVisible && !props.callout,
-          hidden: !isVisible && !props.tooltip,
-        },
-        className,
-      )}
-      ref={attentionEl}>
+    <div data-testid="attention-el" className={containerClasses} ref={attentionEl}>
       <div
         role={props.role === '' ? undefined : props.tooltip ? 'tooltip' : 'img'}
         aria-label={ariaLabel === '' ? undefined : ariaLabel ?? defaultAriaLabel()}
```
Comment (BalbinaK) [inline packages/attention/src/component.tsx:178]: It's not introduced in this PR but the translations are not working after the i18n calls were moved to `/packages/_helpers/attention.tsx`.

File: packages/attention/src/component.tsx (hunk 3 above)
Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx:190]: Updated in these commits: [8e6c70c](https://github.com/warp-ds/react/pull/274/commits/8e6c70c66b1ac7e5e306cd55a0da250a9b765f85) and [3b29cbb](https://github.com/warp-ds/react/pull/274/commits/3b29cbb736f59b69888ec013f845e5ccbfc3e67d) 🙌

File: packages/attention/src/component.tsx (hunk 4 above)
Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx:178]: Updated in these commits: [8e6c70c](https://github.com/warp-ds/react/pull/274/commits/8e6c70c66b1ac7e5e306cd55a0da250a9b765f85) and [3b29cbb](https://github.com/warp-ds/react/pull/274/commits/3b29cbb736f59b69888ec013f845e5ccbfc3e67d) 🙌

**Review by BalbinaK — APPROVED** [review]
LGTM 🎉

**Pull request merged**